continuous-testing-foundations/dotnet/Pragmatest.Wallets/src/Pragmatest.Wallets.Web/Mappers/WalletMappingProfile.cs

```diff
@@ -9,7 +9,7 @@ public class WalletMappingProfile : Profile
         public WalletMappingProfile()
         {
             CreateMap<Balance, BalanceResponse>();
-            CreateMap<DepositRequest, Deposit>();
+            CreateMap<WithdrawalRequest, Deposit>();
```

Comment:
I think this change is related to the others but is a mistake

Reply:
Fixed